src/main/kotlin/racingcar/Wheel.kt

```diff
@@ -0,0 +1,26 @@
+package racingcar
+
+class Wheel {
+    fun nextLocation(power: Int): Int {
+        if (isInValidPower(power)) {
+            throw IllegalArgumentException("유효하지 않은 power 입니다")
+        }
+        return if (isMovablePower(power)) {
+            1
+        } else {
+            return 0
```

Comment:
return을 빼도 동일한 결과가 나올 것 같네요. 👀
```suggestion
            0
```